gamemodes/terrortown/entities/weapons/weapon_kra_carry.lua

```diff
@@ -353,10 +394,42 @@ if CLIENT then
         AddHook("InputMouseApply", "Krampus_Victim_InputMouseApply_" .. entIdx, function(cmd, x, y, ang)
             if not client:Alive() or client:IsSpec() then return end
 
-            -- Lock view in the center, but facing the same direction as Krampus
-            ang = krampus:EyeAngles()
-            ang.pitch = 0
-            cmd:SetViewAngles(ang)
+            -- If we're being held by the krampus, lock our view in the center but facing the same direction as Krampus
+            -- If they aren't being held then the mouse is basically just disabled, preventing them from moving their camera
+            -- This lock takes effect in the delay after the victim is dropped by the Krampus
+            if client:GetNWBool("KrampusCarryVictim", false) then
+                local currentYaw = client:EyeAngles().yaw
+                local targetYaw = krampus:EyeAngles().yaw
+
+                local speedMult = 0.001
+                local minSpeed = 0.001
+
+                local dir = currentYaw < targetYaw and 1 or -1
+                local difference = math.abs(currentYaw - targetYaw)
```

Comment:
We can cache this method
```suggestion
                local difference = MathAbs(currentYaw - targetYaw)
```